doc/tutorials/cascading.md

```diff
@@ -0,0 +1,135 @@
+## Order of namespaces
+
+This tutorial assumes that you know what [namespaces](/doc/tutorials/namespaces.md) are. We will only be talking about [cascading lookup](/doc/help/elektra-cascading.md) here.
+
+In Elektra, the default order of namespaces is as follows:
+
+ * [spec](https://github.com/ElektraInitiative/libelektra/blob/master/doc/help/elektra-namespaces.md#spec) (contains metadata, e.g. to modify elektra lookup behaviour)
+ * [proc](https://github.com/ElektraInitiative/libelektra/blob/master/doc/help/elektra-namespaces.md#proc) (process-related information)
+ * [dir](https://github.com/ElektraInitiative/libelektra/blob/master/doc/help/elektra-namespaces.md#dir) (directory-related information, e.g. `.git` or `.htaccess`)
+ * [user](https://github.com/ElektraInitiative/libelektra/blob/master/doc/help/elektra-namespaces.md#user) (user configuration)
+ * [system](https://github.com/ElektraInitiative/libelektra/blob/master/doc/help/elektra-namespaces.md#system) (system configuration)
+
+Looking at this order, we can see that if a configuration option isn't specified
+by the user (then it would be in the `user` namespace), it will be loaded from
+the `system` namespace. In this case, the option in the `system` namespace will
+be used if the key hasn't been defined by the user.
+
+```
+$ sudo kdb set system/t/test "hello world"
```

Comment:
Would be better if you use fully conforming key names here, and override as profile.